File: packages/warden-core/test/fixtures/vuln-dotnet/Controllers/CardController.cs
using System.Security.Cryptography;
using Microsoft.EntityFrameworkCore;
public class CardController {
    public byte[] Weak(byte[] d) => MD5.Create().ComputeHash(d);
    public void Find(AppDb db, string id) {
        db.Users.FromSqlRaw("SELECT * FROM Users WHERE Id = " + id);
    }
    public void Run(string name) { System.Diagnostics.Process.Start("report " + name); }
    public string CardNumber { get; set; }
    public string Cvv { get; set; }
}

File: packages/warden-core/test/fixtures/vuln-laravel/app/Http/Controllers/CardController.php
<?php
namespace App\Http\Controllers;
use Illuminate\Support\Facades\DB;
class CardController {
    public function hashPw($pw) { return md5($pw); }
    public function find($id) {
        return DB::select("SELECT * FROM users WHERE id = " . $id);
    }
    public function run($name) { return shell_exec("report " . $name); }
    public function store($card_number, $cvv) {
        DB::table('cards')->insert(['card_number' => $card_number, 'cvv' => $cvv]);
    }
    public function all() { return \App\Models\User::withoutGlobalScopes()->get(); }
}
